### DISCUSSION TITLE
GitHub Copilot Nightly (Copilot X) - VS Code Extension Error

### DISCUSSION THREAD

**Discussion by raffertyuy** [community/community#52684] [Copilot Conversations]
### Select Topic Area

Bug

### Body

So I learned that I need to use VS Code Insiders to use Copilot X.
All is well until I tried settings sync - to stable settings. In my settings, I want everything the to be the same (in this case use GitHub Copilot Nightly for both vscode and vscode-insiders.

Now I'm having an issue with VS Code Insiders. Here is the screenshot and output log
![image](https://user-images.githubusercontent.com/1037626/231634115-19a10227-70ed-474b-b759-5d0b0c6196a5.png)

**Output Log**
```
[ERROR] [default] [2023-04-13T02:37:25.272Z] GitHub Copilot could not connect to server. Extension activation failed: "Extension 'GitHub.copilot-nightly' CANNOT use API proposal: semanticSimilarity.
Its package.json#enabledApiProposals-property declares: inlineCompletionsAdditions, interactive but NOT semanticSimilarity.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api GitHub.copilot-nightly"
```

I am using Windows 11. What I tried (all didn't work):
- [x] Uninstall both VS Code and VS Code Insiders, also cleared `%APPDATA%\Code` and `%USERPROFILE%\.vscode` (and code-insiders/.vscode-insiders)
- [x] Turn off settings sync and turn on
- [x] Reinstall GitHub Copilot Nightly (with different sequences, such as uninstall, turn off settings sync, turn on again; or uninstall, then do another clean uninstallation, etc.)

Any advice on what I should do?

GitHub Copilot Nightly works well on my vscode, issue seems to be isolated to insider.

**Comment by blueset** [comment]
Same here. Got the same error message while running GitHub Copilot Nightly 1.82.19 on Visual Studio Code Insiders 1.78.0 commit e5512217bcfb2b644fe7ad6c465fe0b45d23265b on macOS Monterey 12.5 (21G72)

**Reply by raffertyuy** [reply]
Resolved it for now, see my comment below.

**Reply by blueset** [reply]
Thanks, downgrade worked for me!

**Comment by raffertyuy** [comment]
UPDATE: Thought this resolved it but it didn't... 😭

**STEP 1**: The key is turn off settings sync and delete all data
![image](https://user-images.githubusercontent.com/1037626/231638861-9c17bb8f-36bf-4d99-ae5d-28dbd7493e89.png)

**STEP 2**: Then uninstall both VS Code and VS Code Insiders and clear local data
- `%APPDATA%\Code` and `%USERPROFILE%\.vscode`
- `%APPDATA%\Code-Insiders` and `%USERPROFILE%\.vscode-insiders`

**STEP 3**: Reinstall VS Code _**Insiders**_ and Reconfigure
- Install VS Code Insiders
- Turn on Settings Sync, select **Stable** so that the settings will also sync with VS Code later
- Then reinstall extensions, etc.

**Reply by raffertyuy** [reply]
After all that work, looks like the issue is with the latest nightly build `v1.82.19`. Reverted to `v1.82.18` and it works perfectly now.

**Reply by raffertyuy** [reply]
Keeping this issue open for now until the latest build is fixed.

**Reply by ankitbko** [reply]
Reverting it to previous version helped. Thanks a lot.

**Comment by xLeay** [comment]
@raffertyuy Did you got Copilot X from an email ? I'm in the waitlist since it's announced but didn't see anyone who got the access

**Reply by raffertyuy** [reply]
Yes - there was no email. I just went to https://github.com/github-copilot/chat_waitlist_signup again and found out that I already have access. You can try accessing this link to try as well.

Now the key is, this only works on [VS Code Insiders](https://code.visualstudio.com/insiders/) at the moment. And you need to install the GitHub Copilot Nightly extension (not the stable one).

Hence I'm running into all these sync issues :)

**Reply by xLeay** [reply]
To be sure, when you go on the link you gave me, you don't have this 
![image](https://user-images.githubusercontent.com/92150296/231708425-955aac48-69e2-469b-a3d9-539795b771cf.png)
but a message that tells you, you have the access?

**Reply by raffertyuy** [reply]
Ah, ok, this is the screen I'm getting when I click the link.
<img width="361" alt="image" src="https://user-images.githubusercontent.com/1037626/231709705-7177bbe8-be65-47ce-bd4e-eb68a9c54f31.png">

**Reply by xLeay** [reply]
Ok, thanks for you clarity, I think I'll just have to wait a bit more

**Reply by raffertyuy** [reply]
Do you have a paid subscription to Copilot? I believe that's the first step before getting into Copilot X.

**Reply by xLeay** [reply]
I'm a student so I have it for free

**Reply by raffertyuy** [reply]
I see! cool!

**Reply by 0xWeverton** [reply]
@xLeay I've also been waiting for a long time too and I also didn't have access 😞
